.github/blunderbuss.yml

```diff
@@ -12,221 +12,290 @@
+      - "api: workflows"
+    to:
+      - GoogleCloudPlatform/torus-dpe
+  - labels:
+      - "api: asset"
+      - "api: batch"
+      - "api: compute"
+      - "api: cloudkms"
+      - "api: iam"
+      - "api: kms"
+      - "api: privateca"
+      - "api: recaptchaenterprise"
+      - "api: secretmanager"
+      - "api: securitycenter"
+    to:
+      - GoogleCloudPlatform/dee-infra
+  - labels:
+      - "api: clouderrorreporting"
+      - "api: cloudtrace"
+      - "api: container"
+      - "api: logging"
+      - "api: monitoring"
+      - "api: trace"
+    to:
+      - GoogleCloudPlatform/dee-platform-ops
+  - labels:
+      - "api: datacatalog"
+      - 'api: dataflow'
+      - "api: datalabeling"
+      - "api: dataproc"
+      - "api: dialogflow"
+      - "api: enterpriseknowledgegraph"
+      - "api: language"
+      - "api: ml"
+      - "api: notebooks"
+      - "api: optimization"
+      - "api: people-and-planet-ai"
+      - "api: speech"
+      - "api: talent"
+      - "api: texttospeech"
+      - "api: translate"
+      - "api: vision"
+    to:
+      - GoogleCloudPlatform/dee-data-ai
+
+  # AppEco teams
+  - labels:
+      - "api: cloudsql"
+    to:
+      - GoogleCloudPlatform/infra-db-dpes
+  - labels:
+      - "api: bigtable"
+      - "api: datastore"
+      - "api: firestore"
+    to:
+      - GoogleCloudPlatform/cloud-native-db-dpes
+  - labels:
+      - "api: spanner"
+    to:
+      - GoogleCloudPlatform/api-spanner-python
+  - labels:
+      - "api: storage"
+    to:
+      - GoogleCloudPlatform/cloud-storage-dpes
+  - labels:
+      - "api: pubsub"
+      - "api: pubsublite"
+    to:
+      - GoogleCloudPlatform/api-pubsub-and-pubsublite
+  - labels:
+      - "api: composer"
+    to:
+      - GoogleCloudPlatform/cloud-dpes-composer
+
+  # AppEco individuals
+  - labels:
+      - "api: bigquery"
+    to:
+      - shollyman
+  - labels:
+      - "api: datascienceonramp"
+    to:
+      - leahecole
+      - bradmiro
+
+  # Self-service teams
+  - labels:
+      - "api: contentwarehouse"
+      - "api: documentai"
+    to:
+      - GoogleCloudPlatform/document-ai-samples-contributors
+  - labels:
+      - "api: dlp"
+    to:
+      - GoogleCloudPlatform/googleapis-dlp
+  - labels:
+      - "api: functions"
+    to:
+      - GoogleCloudPlatform/functions-framework-google
+  - labels:
+      - "api: billingbudgets"
+      - "api: cloudbilling"
+    to:
+      - GoogleCloudPlatform/billing-samples-maintainers
+  - labels:
+      - "api: healthcare"
+    to:
+      - GoogleCloudPlatform/healthcare-life-sciences
+  - labels:
+      - "api: iot"
+      - "api: cloudiot"
+    to:
+      - GoogleCloudPlatform/api-iot
+  - labels:
+      - "api: retail"
+    to:
+      - GoogleCloudPlatform/cloud-retail-team
+
+  # Self-service individuals
+  - labels:
+      - "api: auth"
+    to:
+      - arithmetic1728
+
 assign_issues:
   - GoogleCloudPlatform/python-samples-owners
 
 assign_prs:
   - GoogleCloudPlatform/python-samples-owners
+
```

Comment:
Minor nit: probably makes sense to move this comment at the end of assign_prs_by section and above the generic assign_issues and assign_prs.